buku

```diff
@@ -5449,7 +5466,7 @@ POSITIONAL ARGUMENTS:
     power_grp = argparser.add_argument_group(
         title='POWER TOYS',
         description='''    --ai                 auto-import bookmarks from web browsers
-                         Firefox, Chrome, Chromium, Edge
+                         Firefox, Chrome, Chromium, Edge, Vivaldi
```

Comment:
Please update the README as well.

Reply:
Done

Reply:
Please maintain the same order here:

    Firefox, Chrome, Chromium, Vivaldi, Edge